.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: build
+
+on:
+  push:
+    branches: [ main, dev ]
```

Comment:
Utils doesn't have "dev" and "certification" branches, no need to specify it here (+ codeql files)

Reply:
fixed 48d5ef7d58db639b69f68724f5506c16ff48ac30